components/Storyblok/SbEmbed.tsx

```diff
@@ -0,0 +1,27 @@
+import { storyblokEditable} from '@storyblok/react/rsc';
+import { Embed } from '@/components/Embed';
+import { type WidthType } from '../WidthBox';
+import { type PaddingType } from '@/utilities/datasource';
+
+type SbEmbedProps = {
+  blok: {
+    _uid: string;
+    src?: string;
+    content?: string;
+    boundingWidth?: 'site' | 'full';
+    width?: WidthType;
+    spacingTop?: PaddingType;
+    spacingBottom?: PaddingType;
+  };
+};
+
+export const SbEmbed = ({ blok, blok: { _uid } }:SbEmbedProps) => {
+
+  return (
+    <Embed
+      {...storyblokEditable}
```

Comment:
```suggestion
      {...storyblokEditable(blok)}
```
If we pass in `blok` here, you can click on the embedded content in the visual editor to bring you into the component editing sidebar 😄

Reply:
whoops, thanks.